metrotranscom/doorway
# fix: new partners logo and updated css

## Issue Overview

This PR addresses [#issue](https://github.com/metrotranscom/doorway/issues/393)


## Description
This switches the partner's site header logo to the new SVG
This matches the partner's site header styling to HBA partner's site styling https://partners.housingbayarea.org/

## How Can This Be Tested/Reviewed?
Open HBA partner site and doorway partner site, you should see that the styling on the site headers matches up 

You should also note that the new logo is used for doorway partners

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
